usaspending_api/references/tests/test_financial_spending.py

```diff
@@ -49,13 +49,25 @@ def financial_spending_data(db):
 def test_award_type_endpoint(client, financial_spending_data):
     """Test the award_type endpoint."""
 
-    resp = client.get('/api/v2/financial_spending/object_class/?fiscal_year=2017&funding_agency_id=1')
+    resp = client.get('/api/v2/financial_spending/major_object_class/?fiscal_year=2017&funding_agency_id=1')
     print(resp.data)
     assert resp.status_code == status.HTTP_200_OK
     assert len(resp.data['results']) == 2
 
     # make sure resp.data['results'] contains a 3000 value
-    assert resp.data['results'][1]['obligated_amount'] == "3000.00"
+    assert (resp.data['results'][1]['obligated_amount'] == "3000.00" or resp.data['results'][0]['obligated_amount'] == "3000.00")
+
+    # check for bad request due to missing params
+    resp = client.get('/api/v2/financial_spending/object_class/')
+    assert resp.status_code == status.HTTP_400_BAD_REQUEST
+
+    resp = client.get('/api/v2/financial_spending/object_class/?fiscal_year=2017&funding_agency_id=1&major_object_class_code=10')
+    print(resp.data)
```

Comment:
Im removing them!  fyi, they dont print anything unless the code fails or the explicit flag is raised.